packages/DatePicker/src/components/ShortcutPanel/ShortcutPanel.styles.js

```diff
@@ -77,6 +77,12 @@ export const containerStyles = css`
     margin: 0;
     overflow: hidden;
     padding: 5px ${tokens.spaceLg} ${tokens.spaceSm} ${tokens.spaceLg};
+
+    /* prevent acl-ui3 label style overrides */
+    input[type="radio"] + label {
+      display: flex;
+      margin: auto;
+    }
```

Comment:
I kind of feel we should include a global style in all of our paprika components (or perhaps only the ones that use inputs with labels) to deal with this instead of adding this style rule in specific locations throughout our paprika components. Saying that though inputs/labels won't always need to be presented in a flex layout.... 🤔

Reply:
I don't know what's the best solution. I think we can go with your fix. We can also use `styled.label` here to avoid overrides.
Something like:
```js
const StyledLabel = styled.label`
  ...
  // styles are not going to be override
`;

export const containerStyles = css`
  ...
  StyledLabel {
    ...
    // Styles are going to be override
  }
`;
```

Reply:
ok, yeah i think for now i'll leave it with the css attribute as this is the common approach we are taking. Perhaps we will change again later on